virtcontainers/kata_agent.go

```diff
@@ -986,9 +987,18 @@ func (k *kataAgent) createContainer(sandbox *Sandbox, c *Container) (p *Process,
 		return nil, err
 	}
 
+	request := &grpc.GuestDetailsRequest{
```

Comment:
We already query guest details in `getAndStoreGuestDetails()`. Please just save the secomp results got from there and then we can use it everywhere. By saving it, please add it to `Sandbox` and also save it on disk so that we can access it again after the very first create sandbox call.